emily/handler/src/api/handlers/testing.rs

```diff
@@ -0,0 +1,42 @@
+//! Handlers for testing endpoint endpoints.
+
+use reqwest::StatusCode;
+use serde_json::json;
+use warp::reply::Reply;
+
+use crate::common::error::Error;
+use crate::context::EmilyContext;
+use crate::database::accessors;
+
+/// Wipe databases handler.
+#[utoipa::path(
+    post,
+    operation_id = "wipeDatabases",
+    path = "/testing/wipe",
+    tag = "testing",
+    responses(
+        // TODO(271): Add success body.
+        (status = 204, description = "Successfully wiped databases."),
+        (status = 400, description = "Invalid request body"),
+        (status = 404, description = "Address not found"),
+        (status = 405, description = "Method not allowed"),
+        (status = 500, description = "Internal server error")
+    )
+)]
+pub async fn wipe_databases(context: EmilyContext) -> impl warp::reply::Reply {
+    // Internal handler so `?` can be used correctly while still returning a reply.
+    async fn handler(context: EmilyContext) -> Result<impl warp::reply::Reply, Error> {
+        accessors::wipe_all_tables(&context).await?;
+        Ok(warp::reply::with_status(
+            warp::reply::json(&json!({
+                "message": "wiped database"
+            })),
+            StatusCode::NO_CONTENT,
```

Comment:
I know this is a test endpoint so it "doesn't matter", but still I figured that I should point out that you probably shouldn't bother returning anything with a `204 No Content` response. I'm under the impression that many clients don't attempt to parse the body.

Reply:
Fair point.